include/CMakeLists.txt

```diff
@@ -23,16 +23,12 @@ install(
   FILES openenclave/host_verify.h
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/
   COMPONENT OEHOSTVERIFY)
+install(FILES openenclave/attestation/attester.h
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/attestation/)
 install(
-  FILES openenclave/attestation/plugin.h
+  FILES openenclave/attestation/verifier.h
   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/attestation/
   COMPONENT OEHOSTVERIFY)
-install(FILES openenclave/attestation/sgx/attester.h
-        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/attestation/sgx/)
-install(
-  FILES openenclave/attestation/sgx/verifier.h
-  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/attestation/sgx/
```

Comment:
This file is incorrectly overwritten. Please revert back to commit 837cc4a4158d90283b3fff91f83c8bdf5319d581.